rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -729,17 +729,17 @@ public JobId submit(File job)
     }
 
     @Override
-    public JobId submit(URL job, Map<String, String> variables, Map<String, String> requestHeaderParams)
+    public JobId submit(File job, Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
-        JobIdData jobIdData = null;
-        try {
-            URLConnection urlConnection = job.openConnection();
+        return submit(job, variables, null);
+    }
 
-            for (Map.Entry<String, String> requestHeaderEntry : requestHeaderParams.entrySet()) {
-                urlConnection.addRequestProperty(requestHeaderEntry.getKey(), requestHeaderEntry.getValue());
-            }
-            InputStream is = urlConnection.getInputStream();
-            jobIdData = restApiClient().submitXml(sid, is, variables);
+    @Override
+    public JobId submit(File job, Map<String, String> variables, Map<String, String> genericInfos)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)